README.md

```diff
@@ -5,11 +5,6 @@ _This SDK is currently in preview._
 
 ## Usage ##
```

Comment:
@pierreca @princjef - Do review the README as well and let me know if I need to make any changes.